### DISCUSSION TITLE
v1.10.0

### DISCUSSION THREAD

**Discussion by webknjaz** [ansible-community/ansible-test-gh-action#38] [Announcements]
## What's New

This version of action started posted per-job test summaries inferred from JUnit reports on the GitHub Actions Workflow Summary pages.

Here's an example of that: https://github.com/ansible-community/ansible-test-gh-action/actions/runs/3139054007#summary-8582766319.

**Full Diff**: https://github.com/ansible-community/ansible-test-gh-action/compare/v1.9.0...v1.10.0

<hr /><em>This discussion was created from the release <a href='https://github.com/ansible-community/ansible-test-gh-action/releases/tag/v1.10.0'>v1.10.0</a>.</em>

**Comment by felixfontein** [comment]
Out of curiosity, why pass `--junit` only for sanity tests? (https://github.com/ansible-community/ansible-test-gh-action/blob/main/action.yml#L354-L358)

**Reply by webknjaz** [reply]
Because other subcommands don't have this option (they will fail with "no such option" if you pass `--junit`). They produce the JUnit output unconditionally.
